src/services/guild.service.ts

```diff
@@ -9,7 +9,7 @@ export class GuildService {
     [Constants.Roles.Unverifed]: undefined,
   };
 
-  private _channelCache: Record<string, Maybe<GuildChannel>> = {
+  private _channelCache: Record<string, Maybe<GuildChannel | ThreadChannel>> = {
```

Comment:
Seems unrelated? Or is it because of the recent change … ugh. Should be a separate PR